### PR TITLE
commit message

### PR DESCRIPTION
**Pull request by diamat1997**
- [DEMO LINK](https://diamat1997.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/styles/blocks/clock.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+.clock {
+  display: flex;
+  justify-content: center;
+  position: relative;
+  border: 5px solid rgb(201, 33, 235);
+  border-radius: 50%;
+  width: $clock-size;
+  height: $clock-size;
+
+  &__center {
```
Comment (alexandra-protyanova) [inline src/styles/blocks/clock.scss:11]: let us more variables and mixins for repetitive styles

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/styles/blocks/clock.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    position: absolute;
+    height: 15px;
+    width: 15px;
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/clock.scss:14]: it's also worth to move this into variable

File: src/styles/blocks/clock.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
+
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/clock.scss:1]: no empty lines on the file beginning

File: src/styles/blocks/clock.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+  &__second {
+    position: absolute;
+    animation: around 60s linear infinite;
+    transform-origin: bottom center;
+    width: 3px;
+    height: $clock-hand;
+    background-color: blue;
+    border: 1px transparent;
+    border-radius: $border-radius;
+  }
```
Comment (ivanchuk-yurii) [inline src/styles/blocks/clock.scss:53]: move repetitive styles into mixin or combining selector

**Review by ivanchuk-yurii — CHANGES_REQUESTED** [review]

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
You haven't added any changes since last review

**Pull request closed without merging**